### ISSUE TITLE
Since 1.9.X tree Support is not only on Heating plate

### ISSUE THREAD

**Issue by druckgott** [bambulab/BambuStudio#4197]
### Bambu Studio Version

1.9.1.66

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 10

### Additional system information

Intel(R) Core(TM) i5-4460  CPU @ 3.20GHz   3.20 GHz
8GB Ram

### Printer

P1P

### How to reproduce

Slize something which only should be on heatplate
![grafik](https://github.com/bambulab/BambuStudio/assets/18383738/0857aa8c-3680-4ed9-93bc-243f983f713d)

Older Versions 1.8.x worked, if I slize it there it´s only on heatplate

### Actual results

It do not only put the tree on the heatplate, it´s also in the part

![grafik](https://github.com/bambulab/BambuStudio/assets/18383738/e9baf3e7-4d0f-4d99-b8fc-9f256746ef89)

### Expected results

Only on Heatplate which worked on 1.8.x with exactly the same part

### Project file & Debug log uploads

I can´t upload a zip file because it´s a part which is not public

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by tangketan** [comment]
Yes, Tree slim could have this bug. It's an old bug. It has nothing to do with the "only on buildplate" option. Try "tree organic" instead.

**Comment by druckgott** [comment]
I dont have this on Same parts in oder version. Its only on the. 1.9.xx version
